iohub/display_utils.py

```diff
@@ -50,11 +103,21 @@ def channel_display_settings(
             clim = channel_settings[chan_name]
         else:
             clim = channel_settings["Other"]
+            display_color = color_to_hex("white")
```

Comment:
```suggestion

```
This line seems to be redundant given [L113-114.](https://github.com/czbiohub/iohub/blob/6dec2b247fe9f3468d638513d3a895fde2100cf4/iohub/display_utils.py#L114)